sqldelight-compiler/dialect/src/main/kotlin/app/cash/sqldelight/dialect/api/SelectQueryable.kt

```diff
@@ -21,16 +20,6 @@ class SelectQueryable(
    * which points to that table (Pure meaning it has exactly the same columns in the same order).
    */
   override val pureTable: NamedElement? by lazy {
-    fun List<QueryColumn>.flattenCompounded(): List<QueryColumn> {
```

Comment:
Moved to `FlattenCompounded` with `internal` visibility, for reuse in new class `ReturningQueryable`

Reply:
Names are okay.